src/components/data/PathListAutomation.js

```diff
@@ -0,0 +1,405 @@
+                                                                    "aria-labelledby": labelId,
+                                                                }}
+                                                            />
+                                                        </ListItemIcon>
+                                                        <ListItemText
+                                                            id={labelId}
+                                                            primary={type}
+                                                        />
+                                                    </ListItem>
+                                                );
+                                            })}
+                                    </List>
+                                </Paper>
+                            </Grid>
+                            <Grid item xs>
+                                <Typography variant="body2">
+                                    {t("pathListDestLbl")}
+                                </Typography>
+                                <Field
+                                    id={buildId(
+                                        baseId,
+                                        ids.PATH_LIST_AUTO_DEST_FIELD
+                                    )}
+                                    name="dest"
+                                    required={true}
+                                    component={SaveAsField}
+                                    label={t("pathListSaveLbl")}
+                                    startingPath={startingPath}
+                                />
+                            </Grid>
+                        </Grid>
+                        <Grid
+                            container
+                            direction="row"
+                            justify="flex-end"
+                            alignItems="flex-end"
+                        >
+                            <Grid item>
+                                <Button
+                                    id={buildId(
+                                        baseId,
+                                        ids.PATH_LIST_AUTO_CANCEL_BTN
+                                    )}
+                                    onClick={onCancel}
+                                >
+                                    {i18nCommon("cancel")}
+                                </Button>
+                            </Grid>
+                            <Grid item>
+                                <Button
+                                    id={buildId(
+                                        baseId,
+                                        ids.PATH_LIST_AUTO_DONE_BTN
+                                    )}
+                                    color="primary"
+                                    type="submit"
+                                    onClick={handleSubmit}
```

Comment:
In local dev testing, I was able to double-click the `Done` button and easily make 6 submissions in a row before the dialog closed and the page redirected. In this case it looked like only the first submission succeeded (I think the rest returned file-exists errors), but if the data store is slow, I'm not sure what kind of race-condition issues this could cause.